ui/constants/collections.js

```diff
@@ -1,5 +1,8 @@
+import * as ICONS from 'constants/icons';
+
 // ui
 export const ICON_SIZE = 12;
+export const PLACEHOLDER = __('My Awesome Playlist');
```

Comment:
Minor, but this will only be evaluated once, and will not update when Language is changed.
- The constant should be a regular string, and the `__()` macro moved to the calling side.